diagnostic.md

```diff
@@ -99,7 +118,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+specialDiv will have font-familu of arial and text color of red. Because cascade
 <!-- your answer ends here -->
```

Comment:
`.specialDiv` will have font-family of cursive because of how specific it's inline HTML styling is and will be `color:yellow`, again because its class style declarations are more specific. Cascading would only determine otherwise if `.things` was equally as specific as `.specialDiv`